### PR TITLE
[dreamfort] overhaul the dreamfort blueprint set

### PR DESCRIPTION
**Pull request by myk002**
as per the several pages of posts following http://www.bay12forums.com/smf/index.php?topic=176889.msg8239017#msg8239017

Many thanks to ldog, who has been awesome on the forums helping me refine dreamfort!

depends on DFHack/scripts#254 for stockpile container configuration

### CODE REVIEW

File: docs/guides/quickfort-alias-guide.rst [hunk 1]
```diff
@@ -499,17 +517,23 @@ give10up
...
+artifactonly      prefix
+togglemasterwork  prefix
+toggleartifact    prefix
+================  ===========
+
+``linksonly``, ``maxbins``, ``maxbarrels``, ``nobins``, ``nobarrels``, and
+``nocontainers`` set the named basic properties on stockpiles. ``nocontainers``
+sets bins and barrels to 0, but does not affect wheelbarrows since the hotkeys
+for changing the number of wheelbarrows depend on whether you have the DFHack
+`stockpiles` plugin active. It is better to set the number of wheelbarrows via
```
Comment (lethosor) [inline docs/guides/quickfort-alias-guide.rst:533]: It's `tweak max-wheelbarrow` that changes the UI, actually

**Review by lethosor — APPROVED** [review]
Docs and aliases look good to me (can't say I've tested the entire fort, though, but it sounds like it has been looked at pretty extensively). I'm fine merging, but let me know if you want to wait for other changes.

File: docs/guides/quickfort-alias-guide.rst (hunk 1 above)
Comment (myk002) [reply docs/guides/quickfort-alias-guide.rst]: whoops! fixed. Thanks!

**Comment by myk002** [comment]
ready to merge. if you don't, I'm afraid I'll be refining this PR forever!

**Comment by lethosor** [comment]
Sounds good (but not intending to discourage further changes!)

**Pull request merged**